index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+	<script type="text/javascript" src="adduser.js"></script>
+	<style>
```

Comment:
Put style in separate file of css and add those in html. Don't write whole bunch of css code in html.